### PR TITLE
[CSL-1317] Document section parameter in browse modules

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: tests/modules/test_browse.py [hunk 1]
```diff
@@ -1126,6 +1153,21 @@ def test_get_browse_facet_options_with_valid_fmt_options():
...
+
+    browse = ConstructorIO(VALID_OPTIONS).browse
+    response = browse.get_browse_facet_options(FACET_NAME, { 'section': SECTION })
+
+    assert isinstance(response.get('request'), dict)
+    assert isinstance(response.get('response'), dict)
+    assert isinstance(response.get('result_id'), str)
+    assert isinstance(response.get('response').get('facets'), list)
+    assert response.get('response').get('facets')[0].get('name') == FACET_NAME
+    assert isinstance(response.get('response').get('facets')[0].get('options'), list)
```
Comment (jjl014) [inline tests/modules/test_browse.py:1167]: Same as above regarding facets.

File: tests/modules/test_browse.py [hunk 2]
```diff
@@ -556,6 +556,19 @@ def test_get_browse_groups_with_valid_fmt_options():
...
+    '''Should return a response with a sections'''
+
+    browse = ConstructorIO(VALID_OPTIONS).browse
+    response = browse.get_browse_groups({ 'section':  SECTION })
+
+    assert isinstance(response.get('request'), dict)
+    assert isinstance(response.get('response'), dict)
+    assert isinstance(response.get('result_id'), str)
+    assert isinstance(response.get('response').get('groups'), list)
+    assert isinstance(response.get('request').get('section'), str)
```
[the remark this answers is not in the document]
Comment (esezen) [reply tests/modules/test_browse.py:569]: I started with this and switched to checking the actual value but forgot to change this test. Good catch.

[the remark this answers is not in the document]
Comment (esezen) [reply tests/modules/test_browse.py]: [ceff989](https://github.com/Constructor-io/constructorio-python/pull/37/commits/ceff989b2d1c33f85594c3964366bab6cd85f92b)

File: tests/modules/test_browse.py [hunk 3]
```diff
@@ -975,7 +1002,7 @@ def test_get_browse_results_for_item_ids_with_valid_item_ids_and_hidden_facets()
     assert isinstance(response.get('result_id'), str)
     assert isinstance(response.get('response').get('results'), list)
     assert response.get('request').get('fmt_options').get('hidden_facets') == hidden_facets
-    assert response.get('response').get('facets')[0].get('name') == hidden_facets[0]
+    assert response.get('response').get('facets')[1].get('name') == hidden_facets[0]
```
[the remark this answers is not in the document]
Comment (esezen) [reply tests/modules/test_browse.py]: [ceff989](https://github.com/Constructor-io/constructorio-python/pull/37/commits/ceff989b2d1c33f85594c3964366bab6cd85f92b)

**Review by jjl014 — APPROVED** [review]
🚀 🚢 💥

**Pull request merged**